### DISCUSSION TITLE
Duvida sobre o uso do Flexbox & Grid

### DISCUSSION THREAD

**Discussion by igorpinhh** [frontendbr/forum#1044] [Perguntas e Respostas]
É recomendado/certo criar layouts inteiros apenas usando 1 desses dois camaradas? talvez os dois juntos ou isso é considerado errado? Até que ponto vocês consideram usar o flexbox em seus projetos? usar eles dois em tudo pode me prejudicar ou algo do tipo?

**Comment by tarciozemel** [comment]
![captura de tela de 2018-04-20 19-09-54](https://user-images.githubusercontent.com/133777/39075874-722d1742-44ce-11e8-896e-729f4322866c.png)

**Comment by RatoX** [comment]
#pracegorver Transição de uma imagem sobre diferenças entre Flexbox e CSS Grid que o @tarciozemel postou.

| Flexbox  | CSS Grid |
| ------------- | ------------- |
| *Unidimensional*  (coluna OU linha) | Bidimensional  |
| A parti do conteúdo  | A partir da definição da grid  |
| Distribuição de espaços | Ocupação de espaços pré-definidos |
| Para porções de layout | Para layouts completos |
| Elementos de UI | Layouts mais abrangentes |
| | Requer menos media queries _minmax(), repeat(), auto-fill etc_. |

**Comment by baltazarparra** [comment]
#105 Tem alguns pontos legais aqui, sobre essa dúvida.

**Comment by mahenrique94** [comment]
@igorpinhh não vejo problema em utilizá-los juntos, você pode ter um layout definido com CSS Grid e dentro dos seus blocos utilizar o Flexbox, já fiz isso e vejo gente fazendo.
